openasip/data/ProGe/cvxif_coprocessor.sv.tmpl

```diff
@@ -0,0 +1,157 @@
+/*
```

Comment:
The hardware description files which affect the produced RTL should be licensed more freely with MIT.